### PR TITLE
update Readme

### PR DESCRIPTION
**Pull request by LanaDX**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,39 +1,47 @@
-<!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128532814/16.1.4%2B)
-[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T114539)
-[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
-<!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
+# GridView for Web Forms - How to calculate values on the fly in batch edit mode
```
Comment (albertov05) [inline Readme.md]: Can we change "on the fly" to "dynamically"?

File: Readme.md [hunk 2]
```diff
@@ -1,39 +1,47 @@
...
 
-* **[Default.aspx](./CS/Default.aspx) (VB: [Default.aspx](./VB/Default.aspx))**
-* [Default.aspx.cs](./CS/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/Default.aspx.vb))
-<!-- default file list end -->
-# ASPxGridView - Batch Edit - How to calculate values on the fly
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/t114539/)**
 <!-- run online end -->
 
+This example demonstrates how to create an unbound column (**Sum**) that changes its values based on other column values on the fly in batch edit mode.
```
Comment (albertov05) [inline Readme.md]: See my comment above regarding "on the fly".

File: Readme.md [hunk 3]
```diff
@@ -1,39 +1,47 @@
...
+
+![Grid View - Modified values](calculate-values-on-the-fly-batch-edit-mode.png)
+
+Set the unbound column's [ShowEditorInBatchEditMode](https://docs.devexpress.com/AspNet/DevExpress.Web.GridDataColumnSettings.ShowEditorInBatchEditMode) property to `false` to  make the column read-only in batch edit mode.
```
Comment (albertov05) [inline Readme.md]:
```suggestion
Set the unbound column's [ShowEditorInBatchEditMode](https://docs.devexpress.com/AspNet/DevExpress.Web.GridDataColumnSettings.ShowEditorInBatchEditMode) property to `false` to make the column read-only in batch edit mode.
```

File: Readme.md [hunk 4]
```diff
@@ -1,39 +1,47 @@
...
-<p>You can find detailed steps by clicking below the "Show Implementation Details" link .<br><strong><br>See Also:</strong></p>
-<p><a href="https://www.devexpress.com/Support/Center/p/T114923">ASPxGridView - How to update total summaries on the client side in Batch Edit mode</a><br><a href="https://www.devexpress.com/Support/Center/p/T116925">ASPxGridView - Batch Edit - How to calculate unbound column and total summary values on the fly</a> <br><a href="https://www.devexpress.com/Support/Center/p/T558750">ASPxGridView - Batch Edit - How to change a cell value based on another cell value</a><br><br><strong>ASP.NET MVC Example:</strong><br><a href="https://www.devexpress.com/Support/Center/p/T124603">GridView - Batch Edit - How to calculate values on the fly</a></p>
+Handle the client [BatchEditEndEditing](https://docs.devexpress.com/AspNet/js-ASPxClientGridView.BatchEditEndEditing) event to re-calculate column values based on the changes and call the [SetCellValue](https://docs.devexpress.com/AspNet/js-ASPxClientGridViewBatchEditApi.SetCellValue(visibleIndex-columnFieldNameOrId-value)) method to set the new column value.
```
Comment (albertov05) [inline Readme.md]:
```suggestion
Handle the client [BatchEditEndEditing](https://docs.devexpress.com/AspNet/js-ASPxClientGridView.BatchEditEndEditing) event to recalculate column values based on the changes and call the [SetCellValue](https://docs.devexpress.com/AspNet/js-ASPxClientGridViewBatchEditApi.SetCellValue(visibleIndex-columnFieldNameOrId-value)) method to set the new column value.
```

File: Readme.md (hunk 2 above)
Comment (LanaDX) [reply Readme.md]:
```suggestion
This example demonstrates how to create an unbound column (**Sum**) that changes its values based on other column values dynamically in batch edit mode.
```

File: Readme.md (hunk 1 above)
Comment (LanaDX) [reply Readme.md]:
```suggestion
# GridView for Web Forms - How to calculate values dynamically in batch edit mode
```

**Pull request merged**